### PR TITLE
Added aggregation expression with tests

### PR DESCRIPTION
**Pull request by SND96**

### CODE REVIEW

File: test/expression/test_aggregation.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        super().__init__(*args, **kwargs)
+
+    def test_aggregation_sum(self):
+        columnName = TupleValueExpression(0)
+        aggr_expr = AggregationExpression(
+            ExpressionType.AGGREGATION_SUM,
+            None,
+            columnName
+        )
+        tuple1 = [[1, 2, 3], [2, 3, 4], [3, 4, 5]]
```
Comment (gaurav274) [inline test/expression/test_aggregation.py]: We are moving from this type of tuple to actual video frames. So we should stop merging more code with this kind of test cases.

File: test/expression/test_aggregation.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        self.assertEqual(6, aggr_expr.evaluate(tuple1, None))
+
+    def test_aggregation_count(self):
+        columnName = TupleValueExpression(0)
+        aggr_expr = AggregationExpression(
+            ExpressionType.AGGREGATION_COUNT,
+            None,
+            columnName
+        )
+        tuple1 = [[1, 2, 3], [2, 3, 4], [3, 4, 5]]
```
Comment (gaurav274) [inline test/expression/test_aggregation.py]: We are moving from this type of tuple to actual video frames. So we should stop merging more code with this kind of test cases.

File: test/expression/test_aggregation.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        self.assertEqual(2, aggr_expr.evaluate(tuple1, None))
+
+    def test_aggregation_min(self):
+        columnName = TupleValueExpression(0)
+        aggr_expr = AggregationExpression(
+            ExpressionType.AGGREGATION_MIN,
+            None,
+            columnName
+        )
+        tuple1 = [[1, 2, 3], [2, 3, 4], [3, 4, 5]]
```
Comment (gaurav274) [inline test/expression/test_aggregation.py]: We are moving from this type of tuple to actual video frames. So we should stop merging more code with this kind of test cases.

File: test/expression/test_aggregation.py (hunk 2 above)
Comment (asrayousuf) [reply test/expression/test_aggregation.py]: Is there an existing test with the actual frames that we could refer to?

**Comment by asrayousuf** [comment]
Incorporated the use of batch frames in the expression test cases. Tests for expressions written by `team 12` passes.

**Comment by jarulraj** [comment]
We already have this functionality in the `master` branch from your other PR. Thanks!

**Pull request closed without merging**